config/base/crds/kafka.banzaicloud.io_kafkaclusters.yaml

```diff
@@ -3916,13 +3942,41 @@ spec:
                     description: ExternalListenerConfig defines the external listener
                       config for Kafka
                     properties:
+                      accessMethod:
```

Comment:
would it make sense to rename this to `externalAccessMethod` to denote that it's about exposing kafka externally through this parameter? as opposed to internal access method which already exists via headless service

Reply:
The `ExternalListenerConfig` already denotes that controls the config of exposing Kafka outside the Kubernetes clusters. The `accessMethod` property defines the `how` to expose Kafka outside the Kubernetes cluster.

Reply:
makes sense! I see now. this is actually under externalListener